### PR TITLE
Fixing SBND CAF maker fcl 

### PR DESCRIPTION
**Pull request by jzennamo**
Updated things to call the correct SimChannel label to match the LArG4 migration.

### CODE REVIEW

File: sbncode/PID/Dazzle_module.cc [hunk 1]
```diff
@@ -89,7 +89,7 @@ class Dazzle : public art::EDProducer {
   art::ServiceHandle<art::TFileService> tfs;
   art::ServiceHandle<cheat::ParticleInventoryService> particleInventory;
 
-  art::InputTag fLArGeantLabel, fPFPLabel, fTrackLabel, fCaloLabel, fMCSLabel, fChi2Label, fRangeLabel, fClosestApproachLabel, fStoppingChi2Label;
+  art::InputTag fLArGeantLabel, fSimChannelLabel, fPFPLabel, fTrackLabel, fCaloLabel, fMCSLabel, fChi2Label, fRangeLabel, fClosestApproachLabel, fStoppingChi2Label;
```
Comment (wesketchum) [inline sbncode/PID/Dazzle_module.cc:92]: fLArGeantLabel no longer used and should be removed.

File: sbncode/PID/Razzle_module.cc [hunk 2]
```diff
@@ -83,7 +83,7 @@ class Razzle : public art::EDProducer {
   art::ServiceHandle<art::TFileService> tfs;
   art::ServiceHandle<cheat::ParticleInventoryService> particleInventory;
 
-  art::InputTag fLArGeantLabel, fPFPLabel, fShowerLabel, fShowerSelVarsLabel;
+  art::InputTag fLArGeantLabel, fSimChannelLabel, fPFPLabel, fShowerLabel, fShowerSelVarsLabel;
```
Comment (wesketchum) [inline sbncode/PID/Razzle_module.cc:86]: fLArGeantLabel no longer used and should be removed.

File: sbncode/PID/sbn_pid.fcl [hunk 3]
```diff
@@ -4,6 +4,7 @@ dazzle:
 {
   module_type:          Dazzle
   LArGeantLabel:        "largeant"
```
Comment (wesketchum) [inline sbncode/PID/sbn_pid.fcl:6]: remove LArGeantLabel

File: sbncode/PID/sbn_pid.fcl [hunk 4]
```diff
@@ -28,6 +29,7 @@ razzle:
 {
   module_type:         Razzle
   LArGeantLabel:       "largeant"
```
Comment (wesketchum) [inline sbncode/PID/sbn_pid.fcl:31]: remove LArGeantLabel

**Review by wesketchum — CHANGES_REQUESTED** [review]

**Comment by jzennamo** [comment]
Here is the output CAF file:
`/sbnd/app/users/jaz8600/AugustPrep/Fixes/Checking/DroppingOptT0Finder/reco1_sce_Reco2-20210819T170607.caf.root`

File: sbncode/PID/sbn_pid.fcl (hunk 4 above)
Comment (jzennamo) [reply sbncode/PID/sbn_pid.fcl]: done

File: sbncode/PID/sbn_pid.fcl (hunk 3 above)
Comment (jzennamo) [reply sbncode/PID/sbn_pid.fcl]: done

File: sbncode/PID/Razzle_module.cc (hunk 2 above)
Comment (jzennamo) [reply sbncode/PID/Razzle_module.cc]: done

File: sbncode/PID/Dazzle_module.cc (hunk 1 above)
Comment (jzennamo) [reply sbncode/PID/Dazzle_module.cc]: done

**Review by etyley — CHANGES_REQUESTED** [review]
If we need to point the backtracker to use `simdrift` I think it is much better to change it where it is defined in sbndcode. This will save many headaches down the line if we get the default version working.

**Comment by jzennamo** [comment]
Currently, in SBNDCode it inherits a configuration from LArSim but I have added that configuration in SBNDCode, does this CAF fcl inherit from SBNDCode?

**Comment by etyley** [comment]
The CAF fcl does inherit a bunch of stuff from sbndcode yes, although it looks like it currently uses the standard backtracker rather than the sbnd version which should be changed.

**Review by wesketchum — APPROVED** [review]
Approving. Can update later to get the SBND-defined BackTracker when it is located. But this will allow us to move ahead with test production.

**Comment by jzennamo** [comment]
I added two lingering issues and @gputnam reviewed the output CAF files and said they looked good

**Comment by jzennamo** [comment]
This needs to be paired with https://github.com/SBNSoftware/sbnanaobj/pull/25

**Pull request merged**